Documentation/helm-operator.md

```diff
@@ -111,6 +111,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.enableGrpcMetrics`             | Enable Ceph CSI GRPC Metrics.                                                                                               | `false`                                                   |
 | `csi.enableCSIHostNetwork`          | Enable Host Networking for Ceph CSI nodeplugins.                                                                            | `false`                                                   |
 | `csi.enablePluginSelinuxHostMount`  | Enable Host mount for /etc/selinux directory for Ceph CSI nodeplugins.                                                      | `false`                                                   |
+| `csi.enableCSIEncryption`  | Enable Ceph CSI PVC encryption support.                                                      | `false`                                                   |
```

Comment:
It looks like most of the other rows add padding after the first text. Why not do that here?